include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1088,7 +1088,13 @@ struct OrtApi {
   ORT_API2_STATUS(CreateEnvWithCustomLoggerAndGlobalThreadPools, OrtLoggingFunction logging_function, _In_opt_ void* logger_param, OrtLoggingLevel logging_level,
                   _In_ const char* logid, _In_ const struct OrtThreadingOptions* tp_options, _Outptr_ OrtEnv** out);
 
- #ifdef USE_CUDA
+  /**
+   * Use this API to configure the global thread pool options to be used in the call to CreateEnvWithGlobalThreadPools.
+   * Set denormal as zero.
+   */
+  ORT_API2_STATUS(SetGlobalDenormalAsZero, _Inout_ OrtThreadingOptions* tp_options);
```

Comment:
>ORT_API2_STATUS(SetGlobalDenormalAsZ [](start = 2, length = 36)

New APIs can only be appended, never inserted to preserve ABI compatibility with C# and potentially others

Reply:
Mistake on merging